packages/faustwp-cli/utils/handleTelemetrySubcommand.ts

```diff
@@ -0,0 +1,84 @@
+import chalk from 'chalk';
+import { getCliArgs } from './getCliArgs.js';
+import { userConfig } from './userConfig.js';
+import { styles } from './styles.js';
+
+const TELEMETRY_ENABLED = 'telemetry.enabled';
+
+const logTelemetryStatus = () => {
+  const statusSetting = userConfig.get(TELEMETRY_ENABLED);
+  const status = statusSetting ? chalk.green('Enabled') : chalk.red('Disabled');
+
+  console.log();
+  console.log(`Status: ${status}`);
+  console.log();
+};
+
+const logTelemetryDocsUrl = () => {
+  console.log('Learn more: https://faustjs.org/docs/telemetry');
+};
+
+const logUserPrefsHaveBeenSaved = () => {
+  console.log(
+    styles.success(`Your preference has been saved to ${userConfig.path}`),
+  );
+};
+
+const logTelemetryAlreadyDisabled = () => {
+  console.log(
+    styles.warn(`Faust.js' telemetry collection is already disabled.`),
+  );
+};
+
+/**
+ * Handles the telemetry subcommand.
+ */
+export const handleTelemetrySubcommand = () => {
+  const telemetryEnabled = userConfig.get(TELEMETRY_ENABLED);
+  const args = getCliArgs();
+  const subcommand = args[1];
+
+  switch (subcommand) {
+    case 'enable': {
+      userConfig.set(TELEMETRY_ENABLED, true);
+      console.log(styles.success('Done!'));
```

Comment:
We should call `logUserPrefsHaveBeenSaved()` for when telemetry is enabled also.